qiskit_experiments/curve_analysis/curve_data.py

```diff
@@ -353,6 +356,15 @@ class FitData:
     x_data: np.ndarray
     y_data: np.ndarray
 
+    @deprecate_func(
```

Comment:
What is the difference between this and other removed methods (they have same removal timeline)?

Reply:
There's no difference, but the old `curve_analysis.visualization` classes like `MplDrawSingleCanvas` still use `FitData` and `SeriesDef`, and those are not scheduled to be removed until after 0.6, so I think the easiest solution is to keep them for another cycle. What do you think?

Reply:
That sounds reasonable.